src/app.rs

```diff
@@ -796,6 +803,7 @@ impl App {
       }
       AppEvent::RemoveProc { id } => {
         self.state.procs.retain(|p| p.is_up() || p.id() != *id);
+        self.check_finish(loop_action);
```

Comment:
In the line before we remove only stopped processes, so this check is not needed.